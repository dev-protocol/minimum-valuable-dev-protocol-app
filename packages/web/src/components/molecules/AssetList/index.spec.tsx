import React from 'react'
import { render } from '@testing-library/react'
import { AssetList } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('src/fixtures/dev-for-apps/hooks')

describe(`${AssetList.name}`, () => {
  test('Snapshot', () => {
    const component = render(<AssetList properties={['1', '2', '3']} onPagination={() => {}} enableStake={true} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
