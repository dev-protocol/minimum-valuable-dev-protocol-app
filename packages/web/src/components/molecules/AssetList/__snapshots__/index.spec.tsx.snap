// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetList Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-17dikip-0 dFeHQH"
    >
      <div
        class="hzc567-1 jJDlWb sc-17dikip-1 cROPlx"
      >
        <div
          class="sc-2tazhb-0 floQve"
        />
        <h3
          class="sc-1w8sy4l-2 hzc567-4 gdOONv lfaQrh"
        >
          1-name
        </h3>
        <div
          class="ant-statistic hzc567-0 hzc567-5 bWyUkR keQvYD"
        >
          <div
            class="ant-statistic-title"
          >
            Your Stake
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                5,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="ant-statistic hzc567-0 hzc567-6 bWyUkR jrcjYC"
        >
          <div
            class="ant-statistic-title"
          >
            Total Staked
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                10,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="hzc567-2 hzc567-7 ckRZOi frZjZU"
        >
          <button
            class="ant-btn sc-1ho13tb-0 EDnks"
            type="button"
          >
            <span>
              Stake
            </span>
          </button>
          
          
        </div>
      </div>
      <div
        class="hzc567-1 jJDlWb sc-17dikip-1 cROPlx"
      >
        <div
          class="sc-2tazhb-0 floQve"
        />
        <h3
          class="sc-1w8sy4l-2 hzc567-4 gdOONv lfaQrh"
        >
          2-name
        </h3>
        <div
          class="ant-statistic hzc567-0 hzc567-5 bWyUkR keQvYD"
        >
          <div
            class="ant-statistic-title"
          >
            Your Stake
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                5,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="ant-statistic hzc567-0 hzc567-6 bWyUkR jrcjYC"
        >
          <div
            class="ant-statistic-title"
          >
            Total Staked
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                10,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="hzc567-2 hzc567-7 ckRZOi frZjZU"
        >
          <button
            class="ant-btn sc-1ho13tb-0 EDnks"
            type="button"
          >
            <span>
              Stake
            </span>
          </button>
          
          
        </div>
      </div>
      <div
        class="hzc567-1 jJDlWb sc-17dikip-1 cROPlx"
      >
        <div
          class="sc-2tazhb-0 floQve"
        />
        <h3
          class="sc-1w8sy4l-2 hzc567-4 gdOONv lfaQrh"
        >
          3-name
        </h3>
        <div
          class="ant-statistic hzc567-0 hzc567-5 bWyUkR keQvYD"
        >
          <div
            class="ant-statistic-title"
          >
            Your Stake
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                5,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="ant-statistic hzc567-0 hzc567-6 bWyUkR jrcjYC"
        >
          <div
            class="ant-statistic-title"
          >
            Total Staked
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                
                10,000
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .00
              </span>
            </span>
          </div>
        </div>
        <div
          class="hzc567-2 hzc567-7 ckRZOi frZjZU"
        >
          <button
            class="ant-btn sc-1ho13tb-0 EDnks"
            type="button"
          >
            <span>
              Stake
            </span>
          </button>
          
          
        </div>
      </div>
      <ul
        class="ant-pagination sc-17dikip-2 cMqJJE"
        unselectable="unselectable"
      >
        <li
          aria-disabled="true"
          class="ant-pagination-prev ant-pagination-disabled"
          title="Previous Page"
        >
          <button
            class="ant-pagination-item-link"
            disabled=""
            tabindex="-1"
            type="button"
          >
            <span
              aria-label="left"
              class="anticon anticon-left"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="left"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                />
              </svg>
            </span>
          </button>
        </li>
        <li
          class="ant-pagination-item ant-pagination-item-0 ant-pagination-disabled ant-pagination-item-disabled"
          tabindex="0"
          title="0"
        >
          <a
            rel="nofollow"
          >
            0
          </a>
        </li>
        <li
          aria-disabled="true"
          class="ant-pagination-next ant-pagination-disabled"
          title="Next Page"
        >
          <button
            class="ant-pagination-item-link"
            disabled=""
            tabindex="-1"
            type="button"
          >
            <span
              aria-label="right"
              class="anticon anticon-right"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                />
              </svg>
            </span>
          </button>
        </li>
      </ul>
    </div>
  </div>
</body>
`;
